dpgen/generator/run.py

```diff
@@ -991,6 +992,8 @@ def run_model_devi (iter_index,
         forward_files += ['input.plumed']
        # backward_files += ['output.plumed']
         backward_files += ['output.plumed','COLVAR','dump.0.xyz']
```

Comment:
You don't need to use a parameter to control it. It's fine to append `HILLS` to `backward_files` directly.